### ISSUE TITLE
Ambiguous use of 'postNotifications'

### ISSUE THREAD

**Issue by ianbytchek** [Quick/Nimble#792]
After updating to 8.1.0 from 8.0.7 `postNotifications` no longer compiles when used without explicitly specifying center parameter:

```swift
expect(expression: myAction).to(postNotifications(equal([myNotification]))) // Ambiguous use of 'postNotifications'
expect(expression: myAction).to(postNotifications(equal([myNotification]), from: .default)) // 👍
```

Though, the `from:` is still optional…

**Comment by ikesyo** [comment]
Thanks for filing an issue. That's because of #789. When the second parameter is omitted we can't disambiguate the two overloads.

**Comment by ikesyo** [comment]
Fixed by #793 and just released [v8.1.1](https://github.com/Quick/Nimble/releases/tag/v8.1.1). Sorry for the inconvenience!

**Comment by ianbytchek** [comment]
@ikesyo That was super-quick! Thanks a bunch, all perfect again!
